cli/tests/e2e/targets/dependency_aware/transitive_and_direct/direct_reachable_transitive_unreachable/package-lock.json

```diff
@@ -0,0 +1,70 @@
+{
+    "requires": true,
+    "lockfileVersion": 2,
+    "packages": {
+        "": {
+          "name": "model-viewer-packages",
+          "license": "Apache-2.0",
+          "devDependencies": {
+            "@typescript-eslint/eslint-plugin": "^4.28.3",
+            "@typescript-eslint/parser": "^4.28.3",
+            "clang-format": "^1.5.0",
+            "eslint": "^7.30.0",
+            "eslint-config-google": "^0.14.0",
+            "eslint-plugin-mocha": "^9.0.0",
+            "eslint-plugin-wc": "^1.3.0",
+            "http-server": "^0.12.3",
+            "husky": "^7.0.1",
+            "lerna": "^4.0.0",
+            "typescript": "4.3.5"
+          },
+          "engines": {
+            "node": ">=12.0.0"
+          }
+        },
+        "node_modules/@babel/code-frame": {
+          "version": "7.12.11"
+        }
+    },
+    "dependencies": {
+        "@types/backbone": {
+            "version": "1.3.43",
+            "resolved": "https://registry.npmjs.org/@types/backbone/-/backbone-1.3.43.tgz",
+            "integrity": "sha512-TAoXvN4neAw2rEOQN8mg0Y0oLYgmPoCWtcMW4Rk19vJahcYueK0Yrf/2TZKWM81K/E1QpHKKyhITaU9LwXAATA==",
+            "dev": true,
+            "requires": {
+                "@types/jquery": "*",
+                "@types/underscore": "*"
+            },
+            "dependencies": {
+                "@types/jquery": {
+                    "version": "3.3.1",
+                    "resolved": "https://registry.npmjs.org/@types/jquery/-/jquery-3.3.22.tgz",
+                    "integrity": "sha512-a4JDcIhJhHYnoWCkG3xT2CZxXZeA92JeREESorg0DMQ3ZsjuKF48h7XK4l5Gl2GRa/ItGRpKMT0pyK88yRgqXQ==",
+                    "dev": true,
+                    "requires": {
+                        "@types/sizzle": "*"
+                    }
+                }
+            }
+        },
+        "@types/jquery": {
+            "version": "3.3.0",
+            "resolved": "https://registry.npmjs.org/@types/jquery/-/jquery-3.3.22.tgz",
```

Comment:
this says 3.3.22; that's because you manually edited the file, right?

Reply:
yeah haha